modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/structure_search_results.jspf

```diff
@@ -1,52 +0,0 @@
-<%--
-/**
- * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
- *
- * This library is free software; you can redistribute it and/or modify it under
- * the terms of the GNU Lesser General Public License as published by the Free
- * Software Foundation; either version 2.1 of the License, or (at your option)
- * any later version.
- *
- * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
- * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
- * details.
- */
---%>
-
-<%
-long searchRestrictionClassNameId = ParamUtil.getLong(request, "searchRestrictionClassNameId");
-long searchRestrictionClassPK = ParamUtil.getLong(request, "searchRestrictionClassPK");
-
-StructureSearchTerms searchTerms = (StructureSearchTerms)searchContainer.getSearchTerms();
-
-long[] groupIds = {PortalUtil.getScopeGroupId(request, refererPortletName, true)};
-
-if (showAncestorScopes) {
-	groupIds = PortalUtil.getCurrentAndAncestorSiteGroupIds(groupIds);
-}
-
-if (searchTerms.isSearchRestriction()) {
-	total = DDMStructureLinkLocalServiceUtil.getStructureLinksCount(searchRestrictionClassNameId, searchRestrictionClassPK);
```

Comment:
Hey @pedroqueiroz , the searchRestriction has been moved to where?

Reply:
There is no reference about that [here](https://github.com/rafaprax/liferay-portal/pull/639/commits/6d581c0c526dc22565948eca09ac01e4dbf08057#diff-c334652ad84f4cc039ebfa951de8a6afR627)